src/test/kotlin/uk/gov/justice/digital/hmpps/sentenceplan/integration/CoordinatorControllerTest.kt

```diff
@@ -99,4 +104,59 @@ class CoordinatorControllerTest : IntegrationTestBase() {
         .expectBody<ErrorResponse>()
     }
   }
+
+  @Nested
+  @DisplayName("signPlan")
+  inner class LockPlan {
```

Comment:
Rename inner class to match display name

Reply:
Resolved in https://github.com/ministryofjustice/hmpps-sentence-plan/pull/109/commits/3a12e9b9aad988b79609fe6e0b46e5b4e0bacc0e

Reply:
Resolved in https://github.com/ministryofjustice/hmpps-sentence-plan/pull/109/commits/a4944f6abb945673e3b227b9b4bc801349c84d2b